homeassistant/components/spotify/browse_media.py

```diff
@@ -145,15 +147,46 @@ class UnknownMediaType(BrowseError):
 
 async def async_browse_media(
     hass: HomeAssistant,
-    media_content_type: str,
-    media_content_id: str,
+    media_content_type: str | None,
+    media_content_id: str | None,
     *,
     can_play_artist: bool = True,
-) -> BrowseMedia:
+) -> BrowseMedia | list[BrowseMedia]:
     """Browse Spotify media."""
-    if not (info := next(iter(hass.data[DOMAIN].values()), None)):
-        raise BrowseError("No Spotify accounts available")
-    return await async_browse_media_internal(
+    parsed_url = None
+    info = None
+
+    # Check if caller is requesting the root nodes
+    if media_content_type is None and media_content_id is None:
+        items = []
+        for config_entry_id, info in hass.data[DOMAIN].items():
+            config_entry = hass.config_entries.async_get_entry(config_entry_id)
+            if not config_entry:
+                raise BrowseError("No config entry found for spotify account")
```

Comment:
I don't think that this is possible since you just iterate the dict.

Reply:
If this is for typing, use `assert config_entry is not None`

Reply:
Yeah, it was for typing. Changed to an assert in 79438dc